app/javascript/components/search/results/StudySearchResult.jsx

```diff
@@ -174,6 +174,9 @@ function studyTypeBadge(study) {
   }
 }
 
+let prevNumSearches = -1
+let numSearchSelections = 0
+
 /**
```

Comment:
It will be much clearer if these properties are tracked inside scp-api-metrics along with `numSearches`--these are much more about global page state than any particular search result component.    Accordingly, let's move these into `scp-api-metrics` alongside `numSearches` and also move logSelectSearchResult there too (or another suitable `[X]-metrics.js` file.)

Reply:
Agreed!  Done in b176314.  The refactor had been scoped elsewhere, but it makes sense to do with this update.